R/calculate_standings.R

```diff
@@ -0,0 +1,72 @@
+#' Compute Division Standings and Conference Seeds from Play by Play
+#'
+#' @param pbp nflfastR play-by-play data either loaded via `load_pbp()` or
+#'   computed via `build_nflfastR_pbp()`.
+#' @param playoff_seeds Number of playoff teams per conference. If `NULL` (the
+#'   default), the function will try to split `pbp` in seasons prior 2020 (6
+#'   seeds) and 2020ff (7 seeds). If set to a numeric, it will be used for all
+#'   seasons in `pbp`!
+#' @inheritParams nflseedR::compute_conference_seeds
+#'
+#' @return A tibble with NFL regular season standings
+#' @export
+#'
+#' @examples
+#' \donttest{
+#' pbp <- nflreadr::load_pbp(c(2018, 2021))
+#' calculate_standings(pbp)
+#' }
+calculate_standings <- function(pbp,
+                                tiebreaker_depth = 3,
+                                playoff_seeds = NULL){
+
+  rlang::check_installed("nflseedR", "to compute standings.",
+                         compare = ">=",
```

Comment:
this should be `>` because it's not in 1.0.2 but it's in devel 1.0.2.9000+

Reply:
I actually had > 1.0.2 first but realized it'll work with 1.0.2 as well. I will just miss losses and ties in the output. That's not a big deal I think.